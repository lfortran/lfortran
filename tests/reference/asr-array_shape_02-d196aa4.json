{
    "basename": "asr-array_shape_02-d196aa4",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/array_shape_02.f90",
    "infile_hash": "4ba934bc62b338ae381400a5ccbac953acdf938e36b87038134b372c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-array_shape_02-d196aa4.stderr",
    "stderr_hash": "af7870ea4a4a3795f6cf089c797aaf9fdc32500d1e866d0f797d3b64",
    "returncode": 2
}