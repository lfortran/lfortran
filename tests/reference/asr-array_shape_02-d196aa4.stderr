semantic error: Incompatible shape of array on assignment on dimension 0 (3 and 2)
 --> tests/errors/array_shape_02.f90:2:1
  |
2 | integer :: x(3,2) = reshape([1,2,3,4],[2,2])
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
